tests/contrib/django/django_tests.py

```diff
@@ -848,7 +848,7 @@ def test_request_metrics_301_append_slash(django_elasticapm_client, client):
         **middleware_setting(
             django.VERSION,
             ["elasticapm.contrib.django.middleware.TracingMiddleware", "django.middleware.common.CommonMiddleware"],
-        )
+        ),
```

Comment:
Are these changes from a code-formatter change from a while ago that is only applied as a particular file is updated?

Reply:
Correct. I touched the file so `black` hit it. We don't run `black` on the test suite, a decision made before I joined Elastic that could probably be revisited.